### DISCUSSION TITLE
Contributors: TargetFrameworks - please comment

### DISCUSSION THREAD

**Discussion by paulushub** [vvvv/SVG#1128] [Contributors]
### Description
All the Tests applications listed below run for net6.0, netcoreapp3.1, net462, on Windows (`.github/workflows/runtests.yml`):
* Tests/Svg.Benchmark
* Tests/Svg.UnitTests
* Tests/SvgW3CTestRunner
```xml
<TargetFrameworks>net6.0;netcoreapp3.1;net462</TargetFrameworks>
```

The target of the `SVG-NET` library, however, include netstandard2.0 and netstandard2.1, as shown below:
```xml
<TargetFrameworks>net6.0;netcoreapp3.1;netstandard2.1;netstandard2.0;net462</TargetFrameworks>
```
### Question
Which platforms or use cases are there (or targeted) for the netstandard2.1 and netstandard2.0?
Scanning the issues report, it seems there are mix results on the support of the .NET Standard, with regards to the `System.Drawing` and `System.Drawing.Common`. It seems someone got it working in web environment - ASP.NET  (cannot remember the OS).

**Comment by mrbean-bremen** [comment]
I remember @wieslawsoltes having run the tests under Linux (don't remember which .NET version, must have been an older one), and a lot of the tests failing due to the problems with libgdiplus, I think.
In the long run, it would be helpful to have tests for different systems.
